test/Concurrency/Runtime/async_task_sleep_cancel.swift

```diff
@@ -0,0 +1,110 @@
+// RUN: %target-run-simple-swift(-Xfrontend -enable-experimental-concurrency %import-libdispatch -parse-as-library) | %FileCheck %s --dump-input always
+// REQUIRES: executable_test
+// REQUIRES: concurrency
+// REQUIRES: libdispatch
+
+// rdar://76038845
+// UNSUPPORTED: use_os_stdlib
+// UNSUPPORTED: back_deployment_runtime
+
+import _Concurrency
+// FIXME: should not depend on Dispatch
+import Dispatch
+
+@available(SwiftStdlib 5.5, *)
+@main struct Main {
+  static let pause = 500_000_000 // 500ms
```

Comment:
it might also be good to test the 0 case of a pause

Reply:
Great point, I'm adding it in a subsequent commit.